ocfweb/docs/docs/membership/banning.md

```diff
@@ -1,6 +1,6 @@
 [[!meta title="Banning policy"]]
 
-_As established by the Board of Directors on October 31, 2016._
+_As modified by the Board of Directors on December 3, 2018._
```

Comment:
preferably keep both established and amended, e.g.

"As established by the Board... on October 31, 2016 and amended December 3, 2018."

Reply:
I would be fine on the other hand without the establishment date (c.f. the account policy)